sound_play/scripts/soundplay_node.py

```diff
@@ -99,7 +99,12 @@ def __init__(self, file, volume = 1.0):
 
     def on_stream_end(self, bus, message):
         if message.type == gst.MESSAGE_EOS:
-            self.state = self.STOPPED
+            if self.state == self.LOOPING:
+                rospy.logdebug("looping")
+                self.sound.seek_simple(gst.FORMAT_TIME, gst.SEEK_FLAG_FLUSH, 0)
+                self.sound.set_state(gst.STATE_PLAYING)
+            elif self.state == self.COUNTING:
+                self.stop()
```

Comment:
This diff looks like it's fixing looping behavior; did this make it into the PR by mistake?